ephios/extra/widgets.py

```diff
@@ -72,3 +72,16 @@ def clean(self, value):
             return rrulestr(value, ignoretz=True)
         except (TypeError, KeyError, ValueError) as e:
             raise ValidationError(_("Invalid recurrence rule: {error}").format(error=e)) from e
+
+
+class PreviousCommentWidget(Widget):
+    template_name = "extra/widgets/previous_comments.html"
+
+    def __init__(self, *args, **kwargs):
+        self.comments = kwargs.pop("comments")
+        super().__init__(*args, **kwargs)
+
+    def get_context(self, name, value, attrs):
+        context = super().get_context(name, value, attrs)
+        context["comments"] = self.comments
+        return context
```

Comment:
I think this and the template could live in core, as they are not general prupose supporting code but bound to the data model in core.